src/plugins/unified_search/public/dataview_picker/change_dataview.tsx

```diff
@@ -26,8 +26,8 @@ import {
   EuiToolTip,
 } from '@elastic/eui';
 import type { DataViewListItem } from '@kbn/data-views-plugin/public';
-import { IDataPluginServices } from '@kbn/data-plugin/public';
 import { useKibana } from '@kbn/kibana-react-plugin/public';
+import type { IUnifiedSearchPluginServices } from '../types';
 import type { DataViewPickerPropsExtended } from '.';
```

Comment:
nit: please try not to use import `from '.'`  Let's keep things simple for webpack